docs/_book.yaml

```diff
@@ -10,6 +10,8 @@ upper_tabs:
     other:
     - name: "Guide & Tutorials"
       contents:
+      - title: "Overview"
```

Comment:
Can we name this "qsim and qsimh" to avoid a naming conflict with the "Overview" tab above?

Reply:
We can, although I don't think we can change the path (`/qsim/overview`) to match. Is that OK?

Reply:
Yeah, that's fine. Can leave the filename

Reply:
Done.